src/main/java/com/cloudbees/jenkins/GitHubPushTrigger.java

```diff
@@ -108,7 +108,6 @@ private boolean runPolling() {
 
             public void run() {
                 if (runPolling()) {
-                    String name = " #" + job.getNextBuildNumber();
```

Comment:
Sure, it is always an estimate. IIRC there was some core PR to not even bother reporting `nextBuildNumber` in the build history widget. But `SCMTrigger` (from which this code seems to have been copied) does log it.